samples/language/Namespaces.qs

```diff
@@ -0,0 +1,14 @@
+/// # Sample
+/// Namespaces
+///
+/// # Description
```

Comment:
Could you please mention how to open namespaces in the description?

Also, could you add opened namespaces to the code, maybe using a function/operation from an opened namespace and add comments explaining the code itself. E.g.:
```qsharp
// The following program declares a namespace block named `MyQuantumApp`.
namespace MyQuantumApp {
    // The following `open` directive is used to import all types and callables declared in the
    // Microsoft.Quantum.Arrays namespace.
    open Microsoft.Quantum.Arrays;

    // Rest of the sample...
}
```